run-checks

```diff
@@ -393,6 +393,34 @@ if [ "$STATIC" = 1 ]; then
       echo "$got"
       exit 1
     fi
+
+    if [ -z "${SKIP_GOLANGCI_LINT-}" ]; then
+
+        echo "Checking installation of golangci-lint"
+        if ! command -v golangci-lint > /dev/null ; then
+            echo "ERROR: Cannot find golangci-lint. You need to first install the golangci-lint snap"
+            exit 1
+        fi
+        if snap refresh --list | grep -q golangci-lint; then
+            echo "WARNING: your golangci-lint snap is out of date. The CI will install a fresh version, which may differ from yours."
+        fi
+        if ! snap list golangci-lint | grep -q latest; then
+            echo "WARNING: your golangci-lint snap is not installed from the latest/* channel."
+        fi
```

Comment:
I suppose those are only relevant iff using golangci-lint from a snap which isn't always the case. Perhaps :
```
gcil="$(command -v golangci-lint || true)"
if [ -z "$gcil" ]; then 
    echo "ERROR: Cannot find golangci-lint. You need to first install the golangci-lint"
    exit 1
fi

if [ "${gcil/\/snap\/bin/}" != "$gcil" ]; then
    # installed from snap
    # check refresh
    # check latest
fi

# check built-with-go-version etc.
```

Reply:
Good point. I added your changes. Thanks!